src/init.js

```diff
@@ -136,4 +136,26 @@ const migrateDirectLocalStorageToNextcloudBrowserStorage = () => {
 	BrowserStorage.setItem('localStorageMigrated', 'done')
 }
 
+/**
+ * Clean unrelated to any conversation keys (if token was not defined, when creating entry).
+ */
+const cleanUnrelatedBrowserStorageKeys = () => {
+	const storageKeys = [
+		'audioDisabled_',
+		'videoDisabled_',
+		'showMediaSettings_',
+		'virtualBackgroundEnabled_',
+		'virtualBackgroundType_',
+		'virtualBackgroundBlurStrength_',
+		'virtualBackgroundUrl_',
+	]
+
+	storageKeys.forEach(key => {
+		if (BrowserStorage.getItem(key) !== null) {
+			BrowserStorage.removeItem(key)
+		}
+	})
+}
+
 migrateDirectLocalStorageToNextcloudBrowserStorage()
+cleanUnrelatedBrowserStorageKeys()
```

Comment:
If the invalid keys do no harm and are just "noise" in the storage I would not clear them. While it is true that they should not be there and Talk should not have ever added them I think that, at least in this case, we could just live with the mistake rather than having code that will fix it once and then never run again.

Having said that fine by me if the cleaning code stays.

Reply:
Removed as not critical